package proto_ff;

import "yd_fieldoptions.proto";


message intimacyintimacyattributeDesc
{
	optional int32 type = 1[(yd_fieldoptions.field_cname) = "ID"];
	optional int32 value = 2[(yd_fieldoptions.field_cname) = "值"];
}

message intimacyintimacy
{
	optional int32 lv = 1[(yd_fieldoptions.field_cname) = "等级"];
	optional int32 intimacyMin = 2[(yd_fieldoptions.field_cname) = "亲密度要求"];
	optional int32 intimacyMax = 3[(yd_fieldoptions.field_cname) = "亲密度要求"];
	repeated intimacyintimacyattributeDesc attribute = 4[(yd_fieldoptions.field_cname) = "基础属性", (yd_fieldoptions.field_arysize) = 3];
}


message Sheet_intimacyintimacy
{
	repeated intimacyintimacy intimacyintimacy_List = 1[(yd_fieldoptions.field_arysize)=115];
}

message intimacygift
{
	optional int64 id = 1[(yd_fieldoptions.field_cname) = "物品id"];
	optional int32 moneyType = 2[(yd_fieldoptions.field_cname) = "货币类型"];
	optional int32 price = 3[(yd_fieldoptions.field_cname) = "价格"];
	optional int32 intimacy = 4[(yd_fieldoptions.field_cname) = "亲密度"];
}


message Sheet_intimacygift
{
	repeated intimacygift intimacygift_List = 1[(yd_fieldoptions.field_arysize)=105];
}
